Week1/Day2/Daily Challenge  Lists & Strings/DC_code.py

```diff
@@ -0,0 +1,19 @@
+#Challenge 1
+number = int(input("Enter your number: "))
+length = int(input('Enter the length of the creating list: '))
+multiples_list = []
+for i in range(1,length+1):
+    multiples_list.append(number*i)
+print(multiples_list)
+
+#Challenge 2
+user_string = input("Enter your word: ")
+final_string = ''
+for i in range(len(user_string)):
+    if user_string[i] != user_string[i-1] or i ==0:
```

Comment:
is there a way to shorten this condition?

Reply:
like this?:
user_string = input("Enter your word: ")
final_string = user_string[0]
for i in range(1,len(user_string)):
    if user_string[i] != user_string[i-1]:
        final_string += user_string[i]
print(final_string)